config/struct_namespace.go

```diff
@@ -28,6 +28,8 @@ type NamespaceConfig struct {
 
 	OrderedLabelNames  []string
 	OrderedLabelValues []string
+
+	HasCounterOnlyLabels bool
```

Comment:
Was not sure if you would prefer this information in the config object or to just pass it to `processSource`. For now I put it in the config object.